src/components/CanvasSaveButton/CanvasSaveButton.js

```diff
@@ -0,0 +1,104 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import OLMap from 'ol/Map';
+import Button from '../Button';
+
+const propTypes = {
+  /**
+   * Translation function.
+   */
+  t: PropTypes.func,
+  conf: PropTypes.object.isRequired,
```

Comment:
I dont' think we need a conf property. To be consistent with other use  a `title` property and use `children` property for the icon.

Reply:
this raise eslint 'no-children-prop' rule which advice to pass children between opening and closing tags instead.

Reply:
we use the children property in other component  like Button and there is no warning.

Reply:
I misunderstood, I thought passing children to <Button>. All good I do this :)